### ISSUE TITLE
Pywal CSS support??

### ISSUE THREAD

**Issue by Ay1tsMe** [philc/vimium#4353]
Not sure if there is a way, but is it possible to apply a pywal generated colour scheme to vimium automatically. I don't see any way to automatically change the CSS within a file. The only way I see is to do it manually in the settings. 

Maybe there could be a way to read a CSS colour scheme template and import them as colour variables so the CSS changes on the fly when pywal changes?

**Comment by philc** [comment]
Hey @Ay1tsMe, I've never tried to modify Vimium's configuration programmatically and I'm not sure it's possible. If it is, I would look into using [chrome-cli ](https://github.com/prasmussen/chrome-cli)to open a tab which contains Vimium's preference page, and then inject javascript which adds the custom CSS into the Vimium settings box,and then clicks save.

**Comment by jas2394** [comment]
Hey, I've explored how Vimium inserts user-defined CSS within `<style></style>` tags. Initially, I attempted to use:

```css
@import url( <either to some remote or local file structure>)
```
Here is my exact attempts:
```css
@import url('file:///home/myUserNameHere/.cache/wal/vimium.css');  /* i got css generted my pywal there*/
```
one with remote:
```css
@import url(https://github.com/dracula/vimium/blob/master/vimium-dracula.css)
```

I thought this might seamlessly incorporate external CSS, but unfortunately, it didn't work as expected.

Then, I tried another approach by closing the style tag first and then defining my own tags later, like in user CSS:

```html
</style>
<link rel="stylesheet" href="path/to/your/stylesheet.css">
```

However, this approach also failed to yield the desired result. It seems that the extension diligently treats everything as strings to prevent the injection of malicious code or something along those lines.

Reopening this issue in hopes that someone might steer me in the right direction or confirm if this approach is feasible.
